test-d/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "test-d",
+  "private": true
+}
```

Comment:
`tsd` seems to be built for declaration-only packages, as it looks at the `types` field in `package.json` (i.e., `dist/index.d.ts`, with no way to disable that behaviour it seems. For that reason, I moved it into a separate folder and added a dummy `package.json`.

Reply:
It sounds like, [judging from the README](https://github.com/SamVerschueren/tsd#order-of-operations), that if you pass a path to `tsd`, then it won't look in the `types` field. Do you think that people will try to run `tsd` on its own instead of `yarn test:types`?

Reply:
I wasn't able to get it to work using `--typings` or by passing a path, but apparently the behaviour is different if you specify a `directory` in the config. Seems to work fine now with the test being in the `src` folder.